.eslintrc

```diff
@@ -1,3 +0,0 @@
-{
-    "extends": "eslint-config-nfl"
```

Comment:
I suppose it is a preference of mine. Reduces the amount of files. Keeps settings in one file (if possible).

But I understand that it might be misleading to developers if they don't see those separate files.

If I revert back to individual files I will add them to the `.npmignore`.